docs/use_cases/personalized_search.md

```diff
@@ -1,15 +1,94 @@
 
-your content here
+A notable example is Spotify, which utilized vector embeddings to enhance its music recommendation system. When evaluated on a large-scale dataset, the use of vector embeddings led to a performance improvement of upwards of 10% in session and track recommendation tasks, showcasing a significant boost in user engagement and satisfaction on the platform [(Hansen et al., 2020)](https://doi.org/10.1145/3383313.3412248). So, what's the secret behind this powerful tool? Let's dive in.
+
+## Why are there limits to traditional search methods?
+
+In this digital age, organizations across the globe are grappling with an exponential surge in data. By 2025, the world is projected to have over 175 zettabytes of data, [source](https://www.forbes.com/sites/tomcoughlin/2018/11/27/175-zettabytes-by-2025). This data, both structured and unstructured, is a treasure trove of insights waiting to be unlocked. The key to this treasure chest? Personalized search using vector embeddings.
+
+## What is a vector embedding?
+
+Vector embeddings are revolutionizing the way we search and retrieve information. They work by converting data into a numerical representation, known as a vector. This process allows the search system to consider the semantics, or the underlying meaning, of the data when performing a search.
+
+![Illustration of vector embeddings](../assets/use_cases/personalized_search/embeddings.png)
+
+For instance, consider a user searching for a book on an online store. With traditional keyword-based search, the user would need to know the exact title or author's name. But with vector embeddings, the user could simply describe the book's theme or plot, and the search system would retrieve relevant results. This is because vector embeddings understand the 'meaning' of the query, rather than just matching keywords.
+
+### How does it search?
+
+The power of vector embeddings lies in their ability to quantify the similarity between two vectors. This is done using a distance metric, with cosine similarity being one of the most commonly used metrics. Simply put, cosine similarity measures how close two vectors are to each other, helping to determine how similar two pieces of data are. It returns relevant results even when the exact terms aren't present in the query. This is made possible by the general-purpose nature of vector embeddings, which can represent almost any form of data, from text to images to audio.
+
+When vectorizing unstructured text data, it's crucial to remember that embedding models have a maximum context length. This is the maximum number of words or phrases that can be embedded into a single vector. For many open-source embedding models, this is in the range of 384 to 1024 words or phrases.
```

Comment:
add 1-2 examples from the top of MTEB to make the claim believable..

https://huggingface.co/spaces/mteb/leaderboard
- ada is 13th and has 8k tokens
- basically everything else seems 512 tokens